src/objective-c/GRPCClient/private/NSDictionary+GRPC.h

```diff
@@ -32,9 +32,9 @@
  */
 
 #import <Foundation/Foundation.h>
-
-struct grpc_metadata;
+#include <grpc/grpc.h>
```

Comment:
Including the header just seems simpler than redeclaring the things in it. Plus, it results in shorter code (`grpc_metadata` vs `struct grpc_metadata`).